src/Common/src/Internal/Padding.cs

```diff
@@ -0,0 +1,22 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+
+namespace Internal
```

Comment:
Not for this PR (we want to get coreclr and corefx back in sync as soon as possible), but subsequently, @maryamariyan I think it would be better if we used a different namespace here, System.Threading or something.